packages/in_app_purchase/lib/src/in_app_purchase/purchase_details.dart

```diff
@@ -103,10 +103,16 @@ class PurchaseParam {
   /// Do not pass in a clear text, your developer ID, the user’s Apple ID, or the
   /// user's Google ID for this field.
   /// For example, you can use a one-way hash of the user’s account name on your server.
-  final String applicationUserName;
+  final String? applicationUserName;
 
-  /// The 'sandboxTesting' is only available on iOS, set it to `true` for testing in AppStore's sandbox environment. The default value is `false`.
+  /// @deprecated Use [simulatesAskToBuyInSandbox] instead.
+  ///
+  /// This parameter has no effect and will be removed shortly.
```

Comment:
Shouldn't it have exactly the same effect as simulatesAskToBuyInSandbox? Otherwise you're not really deprecating it, you're breaking it, which seems strictly worse than just removing it entirely.

Reply:
Redocument this and make this the same effect as simulatesAskToBuyInSandbox.